src/examples/redux-concepts.tsx

```diff
@@ -115,7 +119,7 @@ export const ReduxConcepts: React.FC = () => {
             <p>
               User: {authState.user.name} ({authState.user.email})
             </p>
-            <p>Role: {authState.user.role}</p>
+            <p>Role: {authState.user.roles[0]?.name}</p>
```

Comment:
Make more use of conditional chaining and fallbacks in react to avoid breaking of UI, you can put like this for array as well `authState.user.?roles?.[0]?.name`